docs/core/deploying/trim-self-contained.md

```diff
@@ -32,6 +32,39 @@ When the code is indirectly referencing an assembly through reflection, you can
 </ItemGroup>
 ```
 
+### Support for SSL Certificates
+
+Suppose one of the scenarios is to load SSL certificates within an ASP.NET Core app.  We want to ensure that when trimming we also prevent trimming assemblies that will help with loading SSL certificates.
```

Comment:
- Remove double space
- I would be more aggressive with the wording. I feel like "Suppose" is too related to suggesting they may want to do something when the it's more like problem a exists here is a solution.

```suggestion
If your app loads SSL certificates, such as in an ASP.NET Core app, you'll want to ensure that when trimming you prevent trimming assemblies that will help with loading SSL certificates.
```

Reply:
Adding in the suggestion, thank you!